extensions/wikia/SeoCrossLink/SeoCrossLinkHelper.class.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Class SeoCrossLinkHelper
+ */
+class SeoCrossLinkHelper extends WikiaModel {
+
+	const CACHE_TTL = 86400;
+
+	/**
+	 * Show the module only on article pages (for Anon users only)
+	 * @return boolean
+	 */
+	public function canShowModule() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)